src/qml/MessageAlertBubble.qml

```diff
@@ -27,21 +28,70 @@ ListItemWithActions{
     property var messageData: null
     property var account: null //not used but needed to avoid error in logs
 
-    height: errorTxt.height + textTimestamp.height + units.gu(2)
+    readonly property var messageStatus: messageData.textMessageStatus
+
+    readonly property string permanentErrorText: i18n.tr("Oops, there has been an error with the MMS system and this message could not be retrieved. Please ensure Cellular Data is ON and MMS settings are correct, then ask the sender to try again.")
```

Comment:
Please make sure that translation updates also contain the changes to the pot file for Weblate to pick it up.